hydrocron/api/controllers/timeseries.py

```diff
@@ -172,7 +166,7 @@ def format_csv(feature_lower, results: Generator, feature_id, fields):  # noqa:
     return csv, i
 
 
-def lambda_handler(event, context):  # noqa: E501 # pylint: disable=W0613
+def lambda_handler(event):  # noqa: E501 # pylint: disable=W0613
```

Comment:
Pretty sure lambda requires a 2 argument function. Since we're not using `context`, just rename it to `_` which is the python standard way of documenting an unused argument.

Reply:
done